pyproject.toml

```diff
@@ -64,7 +65,7 @@ test-full = [  # like 'test' but adds dependencies for optional features
 	"scenic[test]",       # all dependencies from 'test' extra above
 	"scenic[guideways]",  # for running guideways modules
 	"astor >= 0.8.1",
-	'carla >= 0.9.12; python_version <= "3.8" and (platform_system == "Linux" or platform_system == "Windows")',
+	'carla >= 0.9.12; python_version <= "3.10"',
```

Comment:
Just wondering, why did we change this?

Reply:
I confirmed removing the platform checks breaks installing Scenic with the `test-full` extra on macOS (`pip install ".[test-full]"`), since there is no `carla` wheel for macOS. Better put them back. (Note that we check for the existence of the `carla` package at runtime, so having the install-time checks here doesn't lead to any problems with Scenic expecting `carla` and failing to import it.)